### PR TITLE
Login css

### PR DESCRIPTION
**Pull request by jioneyang**
Clickable events, added a single test but need to add more, and gotta change some CSS for edit page and need to link with booking somehow.

### CODE REVIEW

**Comment by jioneyang** [comment]
I will work on test coverage

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -6,7 +6,20 @@ class HomeController < ApplicationController
 
     def index
         # redirect_to '/users/sign_in'
+        if current_user #check if user is signed in
+            if current_user.membership == "Club Member"
```
[the remark this answers is not in the document]
Comment (chenjoe569) [reply app/controllers/home_controller.rb]: I don't think you need to check if current_user is true or false. The before action authentication process already takes care of that

File: app/controllers/calendars_controller.rb [hunk 2]
```diff
@@ -0,0 +1,85 @@
+class CalendarsController < ApplicationController
+  before_action :set_calendar, only: [:show, :edit, :update, :destroy]
+
+  # GET /calendars
+  # GET /calendars.json
+  def index
+    if current_user #check if user is signed in
+            if current_user.membership == "Club Member"
```
[the remark this answers is not in the document]
Comment (chenjoe569) [reply app/controllers/calendars_controller.rb]: current_user should always be true because of the authentication by devise

**Comment by chenjoe569** [comment]
Try to resolve the two comments and work on code coverage then do the pull request again?

**Comment by matthewsie** [comment]
Delete links to edit events bc that isnt implemented yet.

**Comment by matthewsie** [comment]
Minor note: highlighted current day seems to be based on a different time zone. Fix to correct timezone? or timezone where current user is located.

**Comment by matthewsie** [comment]
Looks good

**Pull request merged**